lib/modules/datasource/go/releases-direct.spec.ts

```diff
@@ -129,19 +129,19 @@ describe('modules/datasource/go/releases-direct', () => {
           {
             gitRef: 'v0.1.0',
             newDigest: 'd73d815ec22c421e7192a414594ac798c73c89e5',
-            releaseTimestamp: '2022-05-15T16:29:42Z',
+            releaseTimestamp: '2022-05-15T16:29:42.000Z',
```

Comment:
why this change? 🤔

Reply:
We're normalizing everything to the milliseconds now